src/stylesheets/components/_accordions.scss

```diff
@@ -35,6 +35,10 @@ $accordion-border: units($theme-accordion-border-width) solid
     color: color("ink");
     text-decoration: none;
   }
+
+  @media (forced-colors: active) {
+    forced-color-adjust: none;
+  }
```

Comment:
# Accordion

Accordion was missing the expand and collapse icons which implied there was collapsed content hidden. By setting `forced-color-adjust: none` we can maintain original styling, which passes color contrast minimums. 

_Current High Contrast / High Contrast Fix: Dark Theme / High Contrast Fix: Light Theme_
![image](https://user-images.githubusercontent.com/25211650/159997887-d92792a6-0a93-4890-ad23-7da358a6491b.png)